doc/reference/misc.md

```diff
@@ -3548,6 +3548,25 @@ Returns the next value in *pq* without popping it from the priority queue like
 ```
 :::
 
+### pqueue-peek
```

Comment:
this should probably be `pqueue-contents`.

Reply:
Thanks. Fixed.